cmake/mbgl.cmake

```diff
@@ -40,9 +40,15 @@ endfunction()
 
 # Run submodule update
 message(STATUS "Updating submodules...")
-execute_process(
-    COMMAND git submodule update --init mapbox-gl-js
-    WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}")
+if(MBGL_PLATFORM STREQUAL "ios")
+    execute_process(
+        COMMAND git submodule update --init --recursive
```

Comment:
Can we explicitly list all submodule paths here? This will prevent us from updating any submodules that we may add in the future that are unrelated to iOS.